vignettes/BiostringsQuickOverview.Rmd

```diff
@@ -0,0 +1,221 @@
+---
+title: "Biostrings Quick Overview"
+author:
+- |
+  Hervé Pagès\
+  Fred Hutchinson Cancer Research Center\
+  Seattle, WA
+date: "`r format(Sys.time(), '%d %B, %Y')`"
+output:
+  html_document:
+  toc: yes
+  df_print: paged
+  number_sections: yes
+---
+
+Most but not all functions defined in the [Biostrings]{.sans-serif}
+package are summarized here.
+
+::: center
+::: {#table:Low_level_manipulation}
```

Comment:
Replace with
```
## Low-level manipulation of `DNAStringSet` and `AAStringSet` objects.
```
so that we get a header that's a link. You can do this with the other tables below plus removing the caption at the bottom at lines 36-39.